daal4py/sklearn/neighbors/_base.py

```diff
@@ -0,0 +1,63 @@
+#
+# *******************************************************************************
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************/
+
+
+import numpy as np
+
+
+def _get_weights(dist, weights):
```

Comment:
Why this function is needed if predict_proba will not be patched?

Reply:
Removed